### DISCUSSION TITLE
{SelectComponent}.Stack

### DISCUSSION THREAD

**Discussion by showduhtung** [mantinedev/mantine#4124] [Feature requests]
>[Checkbox.Group](https://mantine.dev/core/checkbox/), [Switch.Group](https://mantine.dev/core/switch/) and [Radio.Group](https://mantine.dev/core/radio/) components no longer include [Group](https://mantine.dev/core/group/) – orientation, offset and spacing props are no longer supported. This change gives you more freedom on how to organize inputs layout.
[Chip.Group](https://mantine.dev/core/chip/) no longer includes Group – you need to manage layout on your side

The previous SelectComponent.Group component previous props `orientation`, `offset` and `spacing` made using them very easy. I understand the direction is to allow for customized layouts, but having a default vertical option, to me, is typical (especially for checkbox and radios). 

Thus, I'd like to request an additional `SelectComponent.Stack` component.

**Comment by rtivital** [comment]
You can use Stack component for that. For example:

```tsx
<Checkbox.Group>
  <Stack>
    <Checkbox value="1" />
    <Checkbox value="2" />
  </Stack>
</Checkbox.Group>
```

**Reply by showduhtung** [reply]
My mistake, I was under the impression that the .Group component used the Group component under the hood. And thought the logical next step was to allow Stack
